docs/configuration.md

```diff
@@ -78,5 +80,3 @@ transformers: [
   Plugin.Latex({ renderEngine: "katex" }), // specify some options
 ]
 ```
-
-If you'd like to make your own plugins, read the guide on [[making plugins]] for more information.
```

Comment:
why did this get removed?

Reply:
I had some issues with the current Quart branch and the branch before. Git probably removed it while resolving it?